src/Microsoft.Identity.Web/WebAppAuthenticationBuilderExtensions.cs

```diff
@@ -162,5 +162,18 @@ public static class WebAppAuthenticationBuilderExtensions
             return builder;
         }
 
+        /// <summary>
+        /// Ensures that the authority is a v2.0 authority
+        /// </summary>
+        /// <param name="options">OpenIdConnect options read from the config file
+        /// or set by the developer, for which we want to ensure the authority
+        /// is a v2.0 authority</param>
+        internal static void EnsureAuthorityIsV2_0(OpenIdConnectOptions options)
```

Comment:
Maybe the main part of this method and the one in `WebApiAuthenticationBuilderExtensions` can be extracted into `AuthorityHelpers` class.
Something like so:
```csharp
internal static bool EnsureAuthorityIsV2_0(string authority)
{
    authority = authority.Trim().TrimEnd('/');
    if (!authority.EndsWith("v2.0"))
        authority += "/v2.0";
    return authority;
}
```

Then just called like so from this method:
```csharp
...
options.Authority = AuthorityHelpers.EnsureAuthorityIsV2_0(options.Authority);
...
```

Reply:
with EnsureAuthorityIsV2_0 returning a string, I guess? not a bool?

Reply:
Good catch, @jmprieur 👍. I am not perfect. @jennyf19 got it right, though. 😃